### ISSUE TITLE
Add. 14,525, foll. 1–10 | https://bl.vuexistapps.us/manuscript/243 (Part1)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#1061]
### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

### Problems with Physical Description

- The sentence in the printed catalog has been rearranged in the online version. So, the , that appears before (Add. 14,525, foll. 1—10) seems to be unnecessary: "The first quire of a vellum manuscript, about 7 1/4 in. by 5 1/4, (Add. 14,525, foll. 1—10)."

### Categories of Physical Description Problems

- [ ] Whitespace
- [X] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

**Comment by mikey-tang16** [comment]
@wlpotter 
Should hand 2 and hand 3 be combined into one hand, since Wright does not differentiate the script?

**Comment by wlpotter** [comment]
Yes these can be combined per our current conventions (I'm assuming they were initially split since the two additions are written by different hands, but we've decided not to try and parse all of that out for now)

**Comment by mikey-tang16** [comment]
Ok, hand 2&3 combined
